nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const app = express();
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+app.get("/numbers/add", (req, res) => {
+    if(!req.query.first || !req.query.second) {
+        res.status(400).send("first and second query params are required eg: ?first=1&second=2");
+        return;
```

Comment:
Very nice touch to add such a detailed response - makes it very clear on the receiving end on what went wrong 👌🏽  ⭐